### PR TITLE
Migrate the DNS Naming Service from CE-ARQ

### PR DESCRIPTION
**Pull request by spolti**
**Fixes**: #465

### CODE REVIEW

**Comment by spolti** [comment]
please hold, it needs a couple more changes.

**Comment by spolti** [comment]
Done, feel free to merge it.

File: openshift/openshift/src/test/java/org/arquillian/cube/openshift/DnsServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    /**
+     * Creates a RouteList populated Object to test the dns service
+     *
+     * @param number of routes to be created
+     * @return a populated RouteList object
+     */
+    private static RouteList getRouteList(int number) {
+        RouteList routeList = new RouteList();
+        RouteSpec routeSpec;
```
Comment (lordofthejars) [inline openshift/openshift/src/test/java/org/arquillian/cube/openshift/DnsServiceTest.java]: I think you can move this inside `for`

**Comment by lordofthejars** [comment]
Thank you very much for your collaboration, just a minor thing

**Pull request merged**